.github/ISSUE_TEMPLATE/Bug.md

```diff
@@ -5,19 +5,37 @@ about: Report a problem with rclone
 
 <!--
 
-Welcome :-) We understand you are having a problem with rclone; we want to help you with that!
+We understand you are having a problem with rclone; we want to help you with that!
 
-If you've just got a question or aren't sure if you've found a bug then please use the rclone forum:
+**STOP and READ**
+**YOUR POST WILL BE REMOVED IF IT IS LOW QUALITY**:
+Please show the effort you've put in to solving the problem and please be specific.
+People are volunteering their time to help! Low effort posts are not likely to get good answers!
+
+If you've just got a question or suspect you've found a bug then please use the rclone forum:
 
     https://forum.rclone.org/
 
-instead of filing an issue for a quick response.
+with the "Help & Support Template" for a quick response instead of filing an issue on this repo.
```

Comment:
We also have the `Suspected Bug` Template for suspected bugs instead of `Help & Support` which is targeted for general usage questions.

Reply:
I am in favor of removing the `Help & Support Template` detail from the _github_ template. Let forum admins have freedom to improve their templates without need to chase their changes here.

Reply:
Sounds good.